xbmc/rendering/gl/RenderSystemGL.cpp

```diff
@@ -345,8 +345,7 @@ void CRenderSystemGL::CaptureStateBlock()
   glMatrixTexture.Push();
 
   glDisable(GL_SCISSOR_TEST); // fixes FBO corruption on Macs
-  if (glActiveTextureARB)
```

Comment:
completely broken code, but the logic wasn't. this is an extension that may not be there in principle (graneted, highly unlikely). this is why the decision to remove glew is a peculiar one.